pom.xml

```diff
@@ -117,7 +117,7 @@
     <slf4j.version>1.7.7</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
     <antlr.version>4.5</antlr.version>
-    <rxjava.version>1.0.10</rxjava.version>
+    <rxjava.version>1.0.12</rxjava.version>
```

Comment:
1.0.13 is the newest, might be wise to go there directly (if we're upgrading anyway) ?